README.md

```diff
@@ -48,298 +48,400 @@ The FWS Payments contract enables ERC20 token payment flows through "rails" - au
-  - The sum of the lockup across all rails operated by this contract for a specific client must be at most the maximum per-operator lockup specified by the client.
-- Specifies / changes the payment rail validator of payment rails created by this operator.
-- Terminates payment rails.
+An approved operator can perform the following actions:
 
-### Per-Rail Lockup: Streaming and Fixed Buckets
+-   **Create Rails (`createRail`):** Establish a new payment rail from a client to a service provider, specifying the token, payee, and an optional validator.
+-   **Modify Rail Terms (`modifyRailLockup`, `modifyRailPayment`):** Adjust the payment rate, lockup period, and fixed lockup amount for any rail it manages. Any increase in the client's financial commitment is checked against the operator's allowances.
+-   **Execute One-Time Payments (`modifyRailPayment`):** Execute one-time payments from the rail's fixed lockup.
+-   **Settle Rails (`settleRail`):** Trigger payment settlement for a rail to process due payments. As a rail participant, the operator can initiate settlement at any time.
+-   **Terminate Rails (`terminateRail`):** End a payment rail. Unlike clients, an operator can terminate a rail even if the client's account is not fully funded.
 
-Each payment rail requires the user to manage two lockup components:
+### Per-Rail Lockup: The Guarantee Mechanism
 
-- **Payment Stream lockup:** `paymentRate × lockupPeriod` (covers future payment streams)
-- **Fixed lockup:** `lockupFixed` (covers one-time payments)
+Each payment rail requires the client to lock funds to guarantee future payments. This lockup is composed of two distinct components:
 
-The contract always enforces that the sum of these two values is locked for each rail:
+-   **Streaming Lockup (`paymentRate × lockupPeriod`):** A calculated guarantee for rate based payments for a pre-agreed lockup period.
+-   **Fixed Lockup (`lockupFixed`):** A specific amount set aside for one-time payments.
 
-```
-Total Lockup Required = (paymentRate × lockupPeriod) + lockupFixed
-```
+The total lockup for a client's account is the sum of these requirements across *all* their active rails. This total is reserved from their deposited funds and cannot be withdrawn.
+
+#### The Crucial Role of Streaming Lockup: A Safety Hatch, Not a Pre-payment
+
+It is critical to understand that the streaming lockup is **not** a pre-paid account that is drawn from during normal operation. Instead, it functions as a **safety hatch** that can only be fully utilized *after* a rail is terminated.
+
+**1. During Normal Operation (Before Termination)**
 
-- You must ensure both components are sufficiently funded.
-- One-time payments are deducted from `lockupFixed`.
-- Streaming payments are covered by the streaming lockup.
+While a rail is active, the streaming lockup acts as a **guarantee of solvency**, not a direct source of payment.
```

Comment:
I do wonder though how much this matters, this is basically saying that the lockup isn't a "source", it's not a wallet or a pool, it's just a **rule**. Does the mental framing of that matter so much? Because in your illustrations below you show tokens coming out of `funds` but then the lockup also being reduced by the same amount, so it's _6 on one hand, half a dozen on the other_. What you're trying to do with this sentence is make sure the reader's mental model sees it in a particular way, I'm just wondering if this matters at all.

Reply:
What I want to say is that you can not draw from this "streaming lockup" in the normal course of action for providing your service. You can ONLY use it after you terminate a rail and so it is a guarantee of solvency for a pre-agreed number of epochs. Let me add that to the framing.

We've already explained all over the doc that a client/payer can NEVER withdraw locked funds.